css/index.css

```diff
@@ -1,3 +1,426 @@
+	width: 32%;
+	/* border: 1px solid red; */
+
+}
+
+.bottom-content .product {
+	display: inline-block;
+	width: 32%;
+	margin: 0px 10px 0px;
+}
+
+.bottom-content .vision {
+	display: inline-block;
+	width: 32%;
+}
+
+.contact-content {
+	margin-top: 30px;
+}
+footer p {
+	text-align: center;
+}
+
+/*****************QUERIES*******************/
+
+@media screen and (max-width: 800px) {
+    header{
+      
+        display: flex;
+        justify-content:space-around;
+        height: 100%;
+        align-items:center;
+        flex-direction: column;
+        /* width:100%; */
+        
+
+      }  
+
+    .logo{
+        order: -1;
+        margin-bottom:3%;
+        padding-top:5%;
+        
+
+    }
+   nav a{
+       display:flex;
+       align-content: stretch;
+
+       /* justify-content: space-around; */
+        /* margin-right:0%; 
+        margin-left:9%;  */
+
+       
+
+   }
+    nav{
+        margin-top:2%;
+        margin-bottom:5%;
+        display:flex;
+        justify-content: space-around;
+        display:flex;
+       align-content: stretch;
+        /* margin-left:7%; */
+
+        
+        
+    }
+
+    .introimg{
+        position:absolute;
+        visibility:hidden;
+    } 
+    .container{
+        width:95%;
+        padding-left:12%;
+       
+
+    }  
+    .middle-img{
+        /* padding-right:3%; */
+    }
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+}
+
+@media screen and (max-width: 500px) {
+       .container{
+        width:100%;
+        /* padding-left:7%; */
+       
+
+    } 
+    header{
+      
+      display: flex;
+      width:100%;
+
+      height: 100%;
+      
+      flex-direction: column;
+      margin-left:10%;
+      
+    }  
+    .logo{
+        order: -1;
+        padding-bottom:6%;
+        
+
+    }
+    nav {
+        display:flex;
+        flex-direction: column;
+        
+        /* margin-top:2%; */
+        width:150%;
+        margin-left:0px;
+        margin-right:0px;
+    }
+    a{
+       
+        
+        border-top:1px solid black;
+       
+        width:100%;
+        
+        font-size:24px;
+      
+    }
+    a:last-child{
+        border-bottom:1px solid black;
+        padding:3%;
+        width:100%;
+        
+        
+    }
+    a:first-child{
+        width:100%;
+        padding:3%;
+    }
+    a:nth-child(2n){
+        
+        padding:3%;
+    }
+    a:nth-child(3n){
+        
+        padding:3%;
+    }
+    a:nth-child(4n){
+        
+        padding:3%;
+    }
+    a:nth-child(5n){
+        
+        padding:3%;
+    }
+    .introimg{
+       position:absolute;
+       visibility:hidden;
+   }   
+   .top-content {
+	border-top: 3px solid black;
+    width: 100%;
+    margin:2.5%;
+    display:flex;
+    flex-direction: column;
+    /* align-items:flex-start; */
+    align-items: center;
+   }
+   p{
+       font-size:20px;
+       width:100%;
+    
+       
+   }
+   .intro{
+       margin-left:-15%;
+      
+   }
+   .top-content .features{
```

Comment:
To save room you could do

`.features, .about, .services, .product, .vision{
    width: 100%;
}`

or target the parent classes the same way using commas to separate them. It will apply the style to them all.